src/V3AstNodeOther.h

```diff
@@ -1750,6 +1750,13 @@ class AstVar final : public AstNode {
     // @astgen ptr := m_sensIfacep : Optional[AstIface]  // Interface type to which reads from this
     //                                                      var are sensitive
 
+    enum VRand {
+        NONE,  // Not randomizable
+        RAND,  // Has a rand modifier
+        RAND_CYCLIC,  // Has a randc modifier
+        RAND_INLINE  // Not rand/randc, but used in inline random variable control
+    };
```

Comment:
Please change this to a Verilator-style "class VRand" enum, in V3Ast.h.

Reply:
A class can't be a bit-field.

Reply:
This isn't a hot Ast type (unlike V3Const) so I'd prefer to burn the extra byte(ish) and use the standard enum classes, thanks.